readme.md

```diff
@@ -1 +1,64 @@
-# Hello Microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Linters Project
+
+> This is a project that equips you with the necessary knowledge on linters configurations, making and editing a pull request and adding a professional readme to your project.
+
+## Built With
+
+- HTML and CSS
+- Linters
+- VS studio code, GitHub.
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**In order to use this project, you have to create a .githhub/workflow folder on your root directory and add this file 'linters.yml'. Then checkout to your feature branch and compare with your main branch and then check linters for error on your feature branch.**
+
+---
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/DiamondNath07)
+- Twitter: [@twitterhandle](https://twitter.com/@diamondNath07)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/diamond-nathaniel-6b664b245)
+
+## 🤝 Contributing
+
+Hintrc and stylelintrc to project
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- @BertrandConxy
+- www.microverse.com
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please 🙏🏽 make sure that all the links work as expected, we can use `[MIT](./LICENSE)`. First of all, kindly add the LICENSE file to your repository.